File: go/docmost/models/add_space_members_dto.go
package models

// AddSpaceMembersDto from OpenAPI.
type AddSpaceMembersDto struct {
	SpaceId string `json:"spaceId"`
	Role string `json:"role"`
	UserIds []string `json:"userIds,omitempty"`
	GroupIds []string `json:"groupIds,omitempty"`
}

File: c#/src/Docmost.Sdk/Models/AddSpaceMembersDto.cs
namespace Docmost.Sdk.Models;

public class AddSpaceMembersDto
{
    [System.Text.Json.Serialization.JsonPropertyName("spaceId")]
    public string SpaceId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("role")]
    public string Role { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("userIds")]
    public List<string>? UserIds { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("groupIds")]
    public List<string>? GroupIds { get; set; }

}
